src/foam/u2/grid/testdata/HeroModels.js

```diff
@@ -0,0 +1,213 @@
+foam.CLASS({
```

Comment:
Is this needed to use the GridView? If so, shouldn't it be some place other than in a test.
foam/u2/grid is not the appropriate place for tests/demos or testdata. Better place is com.google.foam.demos.u2.grid.